package/Config.in

```diff
@@ -32,4 +32,5 @@ source "$BR2_EXTERNAL_INFIX_PATH/package/libyang-cpp/Config.in"
 source "$BR2_EXTERNAL_INFIX_PATH/package/sysrepo-cpp/Config.in"
 source "$BR2_EXTERNAL_INFIX_PATH/package/rousette/Config.in"
 source "$BR2_EXTERNAL_INFIX_PATH/package/nghttp2-asio/Config.in"
+source "$BR2_EXTERNAL_INFIX_PATH/package/date/Config.in
```

Comment:
I suggest we use consistent naming, as far as possible, and call it `date-cpp`.

Reply:
I was thinking that, but the project is named "date"

Reply:
Yeah I noticed, but plain date is way too generic. We don't want any inadvertent collisions with buildroot, or other unintended misunderstandings.